documentation/src/docs/pravega-concepts.md

```diff
@@ -376,120 +280,108 @@ The following figure depicts the components deployed by Pravega:
-    one of the Stream Segments based on Routing Key.  For any ReaderGroup
-    reading a Stream, each Stream Segment is assigned to one Reader in that
-    ReaderGroup. 
+-   A stream is partitioned into a set of **Stream Segments**. The number of stream
+    segments in a stream can change over time.  Events are written into exactly
+    one of the stream segments based on **Routing Key**.  For any reader group reading a stream, each stream segment is assigned to one reader in that
+    reader group. 
 
--   Each Stream Segment is stored in a combination of Tier1 and Tier2 storage. 
-    The tail of the Segment is stored in Tier1 providing low latency reads and
-    writes.  The rest of the Segment is stored in Tier2, providing high
-    throughput read access with horizontal scalibility and low cost. 
+-   Each stream segment is stored in a combination of **Tier 1** and **Tier 2** storage. 
+    The tail of the segment is stored in Tier 1 providing low latency reads and
+    writes. The rest of the segment is stored in Tier 2, providing high
+    throughput read access with horizontal scalability and low cost. 
 
 ## A Note on Tiered Storage
 
-To deliver an efficient implementation of Streams, Pravega is based on a tiered
-storage model.  Events are persisted in low latency/high IOPS storage  (Tier 1
-Storage) and higher throughput storage (Tier 2 Storage). Writers and Readers are
+To deliver an efficient implementation of streams, Pravega is based on a tiered
+storage model.  Events are persisted in low latency/high IOPS storage (Tier 1
+storage) and higher throughput storage (Tier 2 storage). Writers and readers are
 oblivious to the tiered storage model from an API perspective. 
 
-Pravega is based on an append-only Log data structure.  As Leigh Stewart
+Pravega is based on an append-only **log** data structure.  As Leigh Stewart
 [observed](https://blog.twitter.com/2015/building-distributedlog-twitter-s-high-performance-replicated-log-service),
-there are really three data access mechanisms in a Log:
+there are really three data access mechanisms in a log:
 
 ![State synchroner](img/anatomy.of.log.png) 
 
 All of the write activity, and much of the read activity happens at the tail of
-the log.  Writes are appended to the log and many clients want to read data as
-fast as it arrives in the log.  These two data access mechanisms are dominated
-by the need for low-latency – low latency writes by Writers and near real time
-access to the published data by Readers. 
-
-Not all Readers read from the tail of the log; some Readers want to read
-starting at some arbitrary position in the log.  These reads are known as
-*catch-up reads*.  Access to historical data traditionally was done by batch
+the log.  Writes are appended to the log and many clients tries to read data immediately as it arrives in the log.  These two data access mechanisms are dominated
+by the need for low-latency. The low latency writes by writers provides real time
```

Comment:
I think that you changed the meaning of the sentence here. The sentence is about two types of data accesses (`writes` and `tail reads`) that require low latency. However, in the new sentence it seems that `low latency writes` yield `real-time reads`. I think that is the system (Pravega) the subject that enables low-latency data access, but not writes themselves (if the system is slow, writes are no longer low latency). I would rather keep the old text in this sentence.